network/wsNetwork.go

```diff
@@ -343,6 +343,8 @@ type WebsocketNetwork struct {
 
 	peersLock          deadlock.RWMutex
 	peers              []*wsPeer
+	peersByID          map[crypto.PublicKey]*wsPeer // peersByID maps a Public Identification Key to a given peer
+	peersByIDLock      deadlock.RWMutex
```

Comment:
I only see one place this is used separately from peersLock, so I think we should just use peersLock